osi_common.proto

```diff
@@ -428,3 +428,210 @@ message BaseMoving
     //
     repeated Vector2d base_polygon = 7;
 }
+
+//
+// \brief A description for the steeringwheel.
+//
+message Steeringwheel
+{
+    // Angle of the steeringwheel. 
+    //
+    // Unit in rad: 0=Central (Straight); Left>0; 0>Right.
+    //
+    optional double angle = 1;
+
+    // Angle-speed of the steeringwheel.
+    //
+    // Unit in rad/s: 0=Central (Straight); Left>0; 0>Right.
+    //
+    optional double angular_speed = 2;
+
+    // Torque of the steeringwheel to the hand.
+    //
+    // Unit in Nm: 0=Central (Straight); Left>0; 0>Right.
+    //
+    optional double torque = 3;
+}
+
+//
+// \brief A description for the positions of the pedals.
+//
+//
+message Pedalry
+{
+    // Position of the acceleration-pedal.
+    // 0-100 (percentage of position: Unpressed - fully pressed)
```

Comment:
@CarloVanDriestenBMW : Here you commented:
// Position of the acceleration-pedal.
//
// Unit: [ ]
//
// Range: [0.0, 1.0]
//
// \note 0.0 := unpressed; 1.0 := fully pressed

For me this would be ok.

Reply:
Changed it to 0-1